Assets/Scripts/StaminaBar.cs

```diff
@@ -24,20 +27,13 @@ public class StaminaBar : MonoBehaviour {
 	}
 
     // Move is called each step of the players movement
-    // Returns state of stamina bar (crash or not);
+    // Returns if player can move;
     public bool move()

```

Comment:
please check the cars when you review, this task is about putting in the pickup called stamina boost, when it is picked up it will boost the players stamina, the player having the ability to collect the pickups is a differnt task blocked by this one.